server/routes/teams.ts

```diff
@@ -47,8 +54,12 @@ export default function routes({
     const team = await serviceCatalogueService.getTeam({ teamSlug, withEnvironments: true })
     const products = team.products.map(product => product)
     const components = getComponentsForTeam(team)
+    const componentNames = getComponentNamesForTeam(team)
     const componentList: string[] =
-      components && components.length > 0 ? components.map(component => component.componentName) : []
+      componentNames && componentNames.length > 0 ? componentNames.map(component => component.componentName) : []
+    const displayComponent = {}
+    let upgradeNeeded = false
```

Comment:
Personal opinion this one 🙂  Swap the `let` and the `const` below it so all the `const` are together.